Doc/library/winreg.rst

```diff
@@ -278,6 +278,36 @@ This module offers the following functions:
       isn't.
 
 
+.. function:: GetValue(key, sub_key, value_name, flags=RRF_RT_ANY)
+
+   Retrieves the type and data for a specified value name associated with
+   an open registry key.
+
+   *key* is an already open key, or one of the predefined
+   :ref:`HKEY_* constants <hkey-constants>`.
+
+   *sub_key* is a string that holds the name of the subkey with which the value is
+   associated.  If this parameter is ``None`` or empty, the value will be read from *key*.
+
+   *value_name* is a string indicating the value to query.
+
+   *flags* restrict the data type of value to be queried.
+   It can be one of the ``RRF_RT_*`` constants, optionally combined ``RRF_NOEXPAND``
+   or one of the ``RRF_SUBKEY_*`` with ``|``.
+   For more info, see :ref:`RRF_* constants <rrf-constants>`.
```

Comment:
```suggestion
   *flags* is an integer that restricts the allowed data type of the value
   to be queried. It must be the bitwise OR of one or more of the
   ``RRF_RT_*`` data types. It may also include ``RRF_NOEXPAND`` or one of
   the ``RRF_SUBKEY_*`` constants. For more information, see
   :ref:`RRF_* constants <rrf-constants>`.
```

Reply:
done